Demos/Blazorise.Demo/Pages/Tests/DataGridPage.razor

```diff
@@ -287,9 +287,6 @@
                                     <SelectItem TValue="string" Value="@("D")">Diverse</SelectItem>
                                 </Select>
                             </FilterTemplate>
-                            @code{

```

Comment:
What the hell was this doing here? Did I merge this in? xDD

Reply:
Might be accidentally merged from my own testings. I tend to throw it randomly like that but this time it slipped unnoticed.